avm/res/api-management/service/main.bicep

```diff
@@ -133,7 +133,7 @@ var authorizationServerList = !empty(authorizationServers) ? authorizationServer
 var formattedUserAssignedIdentities = reduce(map((managedIdentities.?userAssignedResourceIds ?? []), (id) => { '${id}': {} }), {}, (cur, next) => union(cur, next)) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
 
 var identity = !empty(managedIdentities) ? {
-  type: (managedIdentities.?systemAssigned ?? false) ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : null)
+  type: (managedIdentities.?systemAssigned ?? false) ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : 'None')
```

Comment:
Sinc the last Bicep CLI update I'am facing a second issue regarding the differenc between 
`'SystemAssigned,UserAssigned'` and `'SystemAssigned, UserAssigned'`

```
Warning BCP036: The property "type" expected a value of type "'None' | 'SystemAssigned' | 'SystemAssigned, UserAssigned' | 'UserAssigned' | null" but the provided value in source declaration "identity" is of type "'None' | 'SystemAssigned' | 'SystemAssigned,UserAssigned' | 'UserAssigned'".
```

Reply:
@JPEasier  I've seen it in a few modules, but I think that the issue is not the space between `SystemAssigned` and `UserAssigned` but the `null` at the end of the expected value. I have no idea under which condition the `null` should be provided.

Reply:
@JPEasier / @krbar - can this be resolved, the PR is approved but this conversation is blocking merge

Reply:
Yeah would be great to not replace one warning with another one :D @JPEasier do you have an overview for which modules you've experienced the issue? Then I guess we could at least set those to null? Does that make sense? :D

Reply:
Resolving your comment @JPEasier as you approved the PR ;)